src/discover/discover.ml

```diff
@@ -11,13 +11,13 @@ let os c =
     | _ -> Other
 
 let from_pkg_config c =
-  let fallback = ([], [ "-lX11" ]) in
+  let fallback = ([], [ "-lXft"; "-lX11" ]) in
   match C.Pkg_config.get c with
   | None -> fallback
-  | Some pc -> (
-      match C.Pkg_config.query pc ~package:"x11" with
+  | Some pc ->
+      match C.Pkg_config.query pc ~package:"xft x11" with
```

Comment:
You're incredibly lucky.  Dune configurator unhelpfully passes "xft x11" as a single argument to pkg_config, as if one would type
```
usr/bin/pkgconf --personality x86_64-pc-linux-gnu --cflags 'xft x11'
```
(notice the single quotes) in a shell.  But pkgconf helpfully understands that this single `xft x11` argument really means `xft` and then `x11`, and produces the correct result.  Phew!

Reply:
I've read the documentation for `dune-configurator` [here](https://ocaml.org/p/dune-configurator/latest/doc/Configurator/V1/Pkg_config/index.html) which suggested the behavior with space separated packages!